Modules/QuestieQuestLog.lua

```diff
@@ -0,0 +1,239 @@
+---@class QuestieQuestLog
+local QuestieQuestLog = QuestieLoader:CreateModule("QuestieQuestLog")
+
+local elvEnabled = IsAddOnLoaded("ElvUI")
+
+function QuestieQuestLog:IsElvSkinningEnabled(E)
+    return E.private.skins.blizzard.enable == true and E.private.skins.blizzard.quest == true
+end
+
+function QuestieQuestLog:ElvSkinFrames()
+    if elvEnabled == false then
+        return
+    end
+
+    local E, L, V, P, G = unpack(ElvUI)
+    local S = E:GetModule('Skins')
+
+    if QuestieQuestLog:IsElvSkinningEnabled(E) == false then
+        return
+    end
+
+    local function LoadSkin()
+        for i = 1, QUESTS_DISPLAYED do
+            local questLogTitle = _G['QuestLogTitle'..i]
+            questLogTitle:Width(320)
+        end
+
+        QuestLogFrame:HookScript('OnShow', function()
+            QuestLogNoQuestsText:ClearAllPoints();
+            QuestLogNoQuestsText:SetPoint("CENTER", QuestLogFrame);
+        end)
+    end
+
+    S:AddCallback('Quest', LoadSkin)
+end
+
+function QuestieQuestLog:InitializeQuestLogLevelBadges()
+    if not Questie.db.char.enableQuestLogLevelBadges then
+        return
+    end
+
+    local numEntries, numQuests = GetNumQuestLogEntries();
+
+    if (numEntries == 0) then return end
+
+    local questIndex, questLogTitle, title, level, _, isHeader, questTextFormatted, questCheck, questCheckXOfs
```

Comment:
Most of these can be inlined in the for loop. Is there a reason to keep them out of the loop?

Also questCheckXOfs is not used and deplaring `local _` doesn't make much sense 😅